### ISSUE TITLE
mpd can not handle escaped whitespaces

### ISSUE THREAD

**Issue by daniauri** [fatg3erman/RompR#127]
I am sure playing with mpd worked before updating rompr 1.59.1 -> 1.60. However I did also update mpd and now I'm not sure what is the root cause here. 

I have two players: mopidy and mpd, running on the same machine. When adding tracks from a local music library to the playlist with mpd player the list is hanging for a while working and then leave the list empty. Adding and playing tracks from the same music collection using mopity does work. I also run romonitor as a system service for each player to scrobble but this might not be relevant for this issue.

The mpd systemlog is throwing the following error when a local music file is added from rompr:

```
Jan 08 17:09:57 mediacenter mpd[1936559]: client: process command "add "local:track:albums/Green%20Day/2004%20-%20American%20Idiot/02%20-%20Green%20Day%20-%20Jesus%20of%20Suburbia.mp3""
Jan 08 17:09:57 mediacenter mpd[1936559]: exception: No such directory
```

It looks like the mpd deamon have issues with "%20" encoding of whitespace.

Full verbose mpd log when adding music to the playlist via rompr:
```
Jan 08 17:09:52 mediacenter mpd[1936559]: client: [23] opened from [fd15:d441:8ff1::d7c]:38262
Jan 08 17:09:52 mediacenter mpd[1936559]: client: [23] process command "password xxxx"
Jan 08 17:09:52 mediacenter mpd[1936559]: client: [23] command returned 0
Jan 08 17:09:52 mediacenter mpd[1936559]: client: [23] process command list
Jan 08 17:09:52 mediacenter mpd[1936559]: client: process command "stop"
Jan 08 17:09:52 mediacenter mpd[1936559]: client: command returned 0
Jan 08 17:09:52 mediacenter mpd[1936559]: client: process command "clear"
Jan 08 17:09:52 mediacenter mpd[1936559]: client: command returned 0
Jan 08 17:09:52 mediacenter mpd[1936559]: client: [23] process command list returned 0
Jan 08 17:09:52 mediacenter mpd[1936559]: client: [23] process command "status"
Jan 08 17:09:52 mediacenter mpd[1936559]: client: [23] command returned 0
Jan 08 17:09:52 mediacenter mpd[1936559]: client: [23] process command "replay_gain_status"
Jan 08 17:09:52 mediacenter mpd[1936559]: client: [23] command returned 0
Jan 08 17:09:52 mediacenter mpd[1936559]: client: [23] closed
Jan 08 17:09:52 mediacenter mpd[1936559]: client: [24] opened from [fd15:d441:8ff1::13f]:43778
Jan 08 17:09:52 mediacenter mpd[1936559]: client: [24] process command "password xxxxx"
Jan 08 17:09:52 mediacenter mpd[1936559]: client: [24] command returned 0
Jan 08 17:09:52 mediacenter mpd[1936559]: client: [24] process command "playlistinfo"
Jan 08 17:09:52 mediacenter mpd[1936559]: client: [24] command returned 0
Jan 08 17:09:52 mediacenter mpd[1936559]: client: [24] closed
Jan 08 17:09:53 mediacenter mpd[1936559]: client: [25] opened from [fd15:d441:8ff1::d7c]:38264
Jan 08 17:09:53 mediacenter mpd[1936559]: client: [25] process command "password xxxx"
Jan 08 17:09:53 mediacenter mpd[1936559]: client: [25] command returned 0
Jan 08 17:09:53 mediacenter mpd[1936559]: client: [25] process command "status"
Jan 08 17:09:53 mediacenter mpd[1936559]: client: [25] command returned 0
Jan 08 17:09:53 mediacenter mpd[1936559]: client: [25] process command "replay_gain_status"
Jan 08 17:09:53 mediacenter mpd[1936559]: client: [25] command returned 0
Jan 08 17:09:53 mediacenter mpd[1936559]: client: [25] closed
Jan 08 17:09:57 mediacenter mpd[1936559]: client: [26] opened from [fd15:d441:8ff1::13f]:43780
Jan 08 17:09:57 mediacenter mpd[1936559]: client: [26] process command "password xxxxx"
Jan 08 17:09:57 mediacenter mpd[1936559]: client: [26] command returned 0
Jan 08 17:09:57 mediacenter mpd[1936559]: client: [26] process command list
Jan 08 17:09:57 mediacenter mpd[1936559]: client: process command "add "local:track:albums/Green%20Day/2004%20-%20American%20Idiot/02%20-%20Green%20Day%20-%20Jesus%20of%20Suburbia.mp3""
Jan 08 17:09:57 mediacenter mpd[1936559]: exception: No such directory
Jan 08 17:09:57 mediacenter mpd[1936559]: client: command returned 3
Jan 08 17:09:57 mediacenter mpd[1936559]: client: [26] process command list returned 3
Jan 08 17:09:57 mediacenter mpd[1936559]: client: [26] process command "status"
Jan 08 17:09:57 mediacenter mpd[1936559]: client: [26] command returned 0
Jan 08 17:09:57 mediacenter mpd[1936559]: client: [26] process command "status"
Jan 08 17:09:57 mediacenter mpd[1936559]: client: [26] command returned 0
```
Adding a track to the mpd playlist using another app (Andorid MALP) is working and also displayed in rompr. 

Full verbose mpd log when adding music to the playlist via MALP:
```
Jan 08 17:38:35 mediacenter mpd[1951283]: opus: libopus 1.3.1
Jan 08 17:38:35 mediacenter mpd[1951283]: sndfile: libsndfile-1.0.31
Jan 08 17:38:35 mediacenter mpd[1951283]: hybrid_dsd: The Hybrid DSD decoder is disabled because it was not explicitly enabled
Jan 08 17:38:35 mediacenter mpd[1951283]: simple_db: reading DB
Jan 08 17:38:35 mediacenter mpd[1951283]: input: Input plugin 'qobuz' is not configured: No Qobuz app_id configured
Jan 08 17:38:35 mediacenter mpd[1951283]: curl: version 7.80.0
Jan 08 17:38:35 mediacenter mpd[1951283]: curl: with OpenSSL/1.1.1m
Jan 08 17:38:35 mediacenter mpd[1951283]: inotify: initializing inotify
Jan 08 17:38:35 mediacenter mpd[1951283]: inotify: watching music directory
Jan 08 17:38:35 mediacenter systemd[1]: Started Music Player Daemon.
Jan 08 17:39:10 mediacenter mpd[1951283]: client: [0] opened from 192.168.1.205:53654
Jan 08 17:39:10 mediacenter mpd[1951283]: client: [0] process command "password "xxxxx""
Jan 08 17:39:10 mediacenter mpd[1951283]: client: [0] command returned 0
Jan 08 17:39:10 mediacenter mpd[1951283]: client: [0] process command "commands"
Jan 08 17:39:10 mediacenter mpd[1951283]: client: [0] command returned 0
Jan 08 17:39:10 mediacenter mpd[1951283]: client: [0] process command "tagtypes"
Jan 08 17:39:10 mediacenter mpd[1951283]: client: [0] command returned 0
Jan 08 17:39:10 mediacenter mpd[1951283]: client: [0] process command "stats"
Jan 08 17:39:10 mediacenter mpd[1951283]: client: [0] command returned 0
Jan 08 17:39:10 mediacenter mpd[1951283]: client: [0] process command "find album "American Idiot""
Jan 08 17:39:10 mediacenter mpd[1951283]: client: [0] command returned 0
Jan 08 17:39:10 mediacenter mpd[1951283]: client: [0] process command "status"
Jan 08 17:39:10 mediacenter mpd[1951283]: client: [0] command returned 0
Jan 08 17:39:10 mediacenter mpd[1951283]: client: [0] process command "currentsong"
Jan 08 17:39:10 mediacenter mpd[1951283]: client: [0] command returned 0
Jan 08 17:39:10 mediacenter mpd[1951283]: client: [0] process command "status"
Jan 08 17:39:10 mediacenter mpd[1951283]: client: [0] command returned 0
Jan 08 17:39:10 mediacenter mpd[1951283]: client: [0] process command "find album "American Idiot""
Jan 08 17:39:10 mediacenter mpd[1951283]: client: [0] command returned 0
Jan 08 17:39:11 mediacenter mpd[1951283]: client: [0] process command "idle"
Jan 08 17:39:11 mediacenter mpd[1951283]: client: [0] command returned 1
Jan 08 17:39:16 mediacenter mpd[1951283]: client: [0] process command "add "albums/Green Day/2004 - American Idiot/03 - Green Day - Holiday.mp3""
Jan 08 17:39:16 mediacenter mpd[1951283]: client: [0] command returned 0
Jan 08 17:39:17 mediacenter mpd[1951283]: client: [0] process command "idle"
Jan 08 17:39:17 mediacenter mpd[1951283]: client: [0] command returned 1
Jan 08 17:39:17 mediacenter mpd[1951283]: client: [0] process command "status"
Jan 08 17:39:17 mediacenter mpd[1951283]: client: [0] command returned 0
Jan 08 17:39:17 mediacenter mpd[1951283]: client: [0] process command "currentsong"
Jan 08 17:39:17 mediacenter mpd[1951283]: client: [0] command returned 0
Jan 08 17:39:17 mediacenter mpd[1951283]: client: [0] process command "playlistinfo 0:1"
Jan 08 17:39:17 mediacenter mpd[1951283]: client: [0] command returned 0
Jan 08 17:39:17 mediacenter mpd[1951283]: client: [0] process command "idle"
Jan 08 17:39:17 mediacenter mpd[1951283]: client: [0] command returned 1
Jan 08 17:39:26 mediacenter mpd[1951283]: client: [0] closed
```
So I think it might not be a mpd issue?

Anyway, the mpd version and plugins:
```➜  music mpd --version
Music Player Daemon 0.23.5 (0.23.5)
Copyright 2003-2007 Warren Dukes <warren.dukes@gmail.com>
Copyright 2008-2021 Max Kellermann <max.kellermann@gmail.com>
This is free software; see the source for copying conditions.  There is NO
warranty; not even MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

Database plugins:
 simple proxy upnp

Storage plugins:
 local udisks nfs curl

Neighbor plugins:
 upnp udisks

Decoders plugins:
 [mad] mp3 mp2
 [mpg123] mp3
 [vorbis] ogg oga
 [oggflac] ogg oga
 [flac] flac
 [opus] opus ogg oga
 [sndfile] wav aiff aif au snd paf iff svx sf voc w64 pvf xi htk caf sd2
 [audiofile] wav au aiff aif
 [dsdiff] dff
 [dsf] dsf
 [hybrid_dsd] m4a
 [faad] aac
 [mpcdec] mpc
 [wavpack] wv
 [openmpt] mptm mod s3m xm it 669 amf ams c67 dbm digi dmf dsm dtm far imf ice j2b m15 mdl med mms mt2 mtm nst okt plm psm pt36 ptm sfx sfx2 st26 stk stm stp ult wow gdm mo3 oxm umx xpk ppm mmcmp
 [modplug] 669 amf ams dbm dfm dsm far it med mdl mod mtm mt2 okt s3m stm ult umx xm
 [mikmod] amf dsm far gdm imf it med mod mtm s3m stm stx ult uni xm
 [sidplay] sid mus str prg P00
 [wildmidi] mid
 [fluidsynth] mid
 [ffmpeg] 16sv 3g2 3gp 4xm 8svx aa3 aac ac3 adx afc aif aifc aiff al alaw amr anim apc ape asf atrac au aud avi avm2 avs bap bfi c93 cak cin cmv cpk daud dct divx dts dv dvd dxa eac3 film flac flc fli fll flx flv g726 gsm gxf iss m1v m2v m2t m2ts m4a m4b m4v mad mj2 mjpe
g mjpg mka mkv mlp mm mmf mov mp+ mp1 mp2 mp3 mp4 mpc mpeg mpg mpga mpp mpu mve mvi mxf nc nsv nut nuv oga ogm ogv ogx oma ogg omg opus psp pva qcp qt r3d ra ram rl2 rm rmvb roq rpl rvc shn smk snd sol son spx str swf tak tgi tgq tgv thp ts tsp tta xa xvid uv uv2 vb vid
vob voc vp6 vmd wav webm wma wmv wsaud wsvga wv wve rtp:// rtsp:// rtsps://
 [gme] ay gbs gym hes kss nsf nsfe rsn sap spc vgm vgz
 [pcm]

Filters:
 libsamplerate soxr

Tag plugins:
 id3tag

Output plugins:
 shout null fifo pipe alsa ao oss openal solaris pipewire pulse jack httpd snapcast recorder

Encoder plugins:
 null vorbis opus lame twolame wave flac

Archive plugins:
 [bz2] bz2
 [zzip] zip
 [iso] iso

Input plugins:
 file io_uring archive alsa qobuz curl ffmpeg nfs mms cdio_paranoia

Playlist plugins:
 extm3u m3u pls xspf asx rss soundcloud flac cue embcue

Protocols:
 file:// alsa:// cdda:// ftp:// ftps:// gopher:// hls+http:// hls+https:// http:// https:// mms:// mmsh:// mmst:// mmsu:// nfs:// qobuz:// rtmp:// rtmpe:// rtmps:// rtmpt:// rtmpte:// rtmpts:// rtp:// rtsp:// rtsps:// scp:// sftp:// smb:// srtp://

Other features:
 avahi dbus udisks epoll icu inotify ipv6 systemd tcp un
```

..and the mpd config file:
```➜  music cat /etc/mpd.conf
# See: /usr/share/doc/mpd/mpdconf.example

user "mpd"
group "audio"
db_file "/var/lib/mpd/mpd.db"
#state_file "/var/lib/mpd/mpdstate"
playlist_directory "/home/media/playlists/"
music_directory "/home/media/music/"
connection_timeout "120"
port "6600"
zeroconf_enabled "yes"
zeroconf_name "MpdHome"
metadata_to_use "artist,album,title,track,name,genre,date,composer,performer,disc"
auto_update "yes"
auto_update_depth "3"
follow_outside_symlinks "yes"
default_permissions "read,add"
password "yyyyyyy@read,add,control"
password "xxxxxxx@read,add,control,admin"
replaygain "album"
replaygain_preamp "0"
volume_normalization "yes"

audio_output {
    type            "fifo"
    name            "Irene"
    path            "/tmp/snapfifo"
    format          "48000:16:2"
    mixer_type      "software"
}

audio_output {
    type            "fifo"
    name            "Daniel"
    path            "/tmp/snapfifo_lq"
    format          "48000:16:2"
    mixer_type      "software"
}

audio_output {
    type                "httpd"
    name                "HTTP Stream"
    encoder             "vorbis"                # optional, vorbis or lame
    port                "8000"
    bind_to_address     "0.0.0.0"               # optional, IPv4 or IPv6
#    quality            "5.0"                   # do not define if bitrate is defined
    bitrate             "128"                   # do not define if quality is defined
    format              "44100:16:1"
    max_clients         "0"                     # optional 0=no limit
}
```
The rompr debug info output is:
```
* **BACKEND INFO**
  * **Version** 1.60
  * **mysql_database** [Redacted]
  * **mysql_host** [Redacted]
  * **mysql_password** [Redacted]
  * **mysql_port** [Redacted]
  * **mysql_user** [Redacted]
  * **lastfm_session_key** [Redacted]
  * **spotify_token** [Redacted]
  * **spotify_token_expires** [Redacted]
* **SERVER INFO**
  * **PHP_SELF** /music/utils/debuginfo.php
  * **SERVER_ADDR** 192.168.1.184
  * **SERVER_NAME** mediacenter.home
  * **SERVER_SOFTWARE** nginx/1.20.2
  * **DOCUMENT_ROOT** /srv/http
  * **HTTP_HOST** mediacenter.home
  * **HTTP_USER_AGENT** Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.110 Safari/537.36
  * **REMOTE_ADDR** 192.168.1.158
  * **SERVER_PORT** 80
  * **REQUEST_URI** /music/utils/debuginfo.php
* **COOKIES**
  * **skin** desktop
  * **player-64616e** 14
  * **player-61646d696e** 15
  * **JSESSIONID** 9FFBFB0810E819F3BD4A1B5A9D6D93BD
  * **currenthost** Daniel
  * **player_backend** mpd
* **PHP INFO**
  * **Version** 8.0.14
  * **mbstring** 8.0.14
  * **PDO** 8.0.14
  * **pdo_mysql** 
  * **pdo_sqlite** 8.0.14
  * **curl** 8.0.14
  * **date** 8.0.14
  * **fileinfo** 8.0.14
  * **json** 8.0.14
  * **SimpleXML** 8.0.14
  * **GD** 8.0.14
  * **GD Info** GD Version=2.3.3, FreeType Support=1, FreeType Linkage=with freetype, GIF Read Support=1, GIF Create Support=1, JPEG Support=1, PNG Support=1, WBMP Support=1, XPM Support=1, XBM Support=1, WebP Support=1, BMP Support=1, TGA Read Support=1, JIS-mapped Japanese Font Support=
  * **ImageMagick** Installed
  * **date.timezone** global_value=, local_value=, access=7
  * **default_charset** global_value=UTF-8, local_value=UTF-8, access=7
  * **default_socket_timeout** global_value=60, local_value=60, access=7
  * **display_errors** global_value=, local_value=, access=7
  * **error_log** global_value=, local_value=, access=7
  * **error_reporting** global_value=22527, local_value=22527, access=7
  * **file_uploads** global_value=1, local_value=1, access=4
  * **log_errors** global_value=1, local_value=1, access=7
  * **max_execution_time** global_value=1800, local_value=1800, access=7
  * **mbstring.language** global_value=neutral, local_value=neutral, access=7
  * **memory_limit** global_value=128M, local_value=128M, access=7
  * **pdo_mysql.default_socket** [NOT SET]
  * **session.use_cookies** global_value=1, local_value=1, access=7
  * **upload_tmp_dir** global_value=, local_value=, access=4
* **PLAYER INFORMATION**
  * **Connection Status** Connection Successful
  * **MPD Interface Version** 0.23.5
  * **tagtype** Artist, Album, Title, Track, Name, Genre, Date, Composer, Performer, Disc
  * **Commands** add, addid, addtagid, albumart, binarylimit, channels, clear, clearerror, cleartagid, close, commands, config, consume, count, crossfade, currentsong, decoders, delete, deleteid, delpartition, disableoutput, enableoutput, find, findadd, getfingerprint, getvol, idle, kill, list, listall, listallinfo, listfiles, listmounts, listpartitions, listplaylist, listplaylistinfo, listplaylists, load, lsinfo, mixrampdb, mixrampdelay, mount, move, moveid, moveoutput, newpartition, next, notcommands, outputs, outputset, partition, password, pause, ping, play, playid, playlist, playlistadd, playlistclear, playlistdelete, playlistfind, playlistid, playlistinfo, playlistmove, playlistsearch, plchanges, plchangesposid, previous, prio, prioid, random, rangeid, readcomments, readmessages, readpicture, rename, repeat, replay_gain_mode, replay_gain_status, rescan, rm, save, search, searchadd, searchaddpl, seek, seekcur, seekid, sendmessage, setvol, shuffle, single, stats, status, stop, subscribe, swap, swapid, tagtypes, toggleoutput, unmount, unsubscribe, update, urlhandlers, volume
  * **URL Handlers** alsa://
  * **URL Handlers** cdda://
  * **URL Handlers** ftp://
  * **URL Handlers** ftps://
  * **URL Handlers** gopher://
  * **URL Handlers** hls+http://
  * **URL Handlers** hls+https://
  * **URL Handlers** http://
  * **URL Handlers** https://
  * **URL Handlers** mms://
  * **URL Handlers** mmsh://
  * **URL Handlers** mmst://
  * **URL Handlers** mmsu://
  * **URL Handlers** nfs://
  * **URL Handlers** rtmp://
  * **URL Handlers** rtmpe://
  * **URL Handlers** rtmps://
  * **URL Handlers** rtmpt://
  * **URL Handlers** rtmpte://
  * **URL Handlers** rtmpts://
  * **URL Handlers** rtp://
  * **URL Handlers** rtsp://
  * **URL Handlers** rtsps://
  * **URL Handlers** scp://
  * **URL Handlers** sftp://
  * **URL Handlers** smb://
  * **URL Handlers** srtp://
  * **plugin** mad, mpg123, vorbis, oggflac, flac, opus, sndfile, audiofile, dsdiff, dsf, faad, mpcdec, wavpack, openmpt, modplug, mikmod, sidplay, wildmidi, fluidsynth, ffmpeg, gme, pcm
  * **suffix** mp3, mp2, mp3, ogg, oga, ogg, oga, flac, opus, ogg, oga, wav, aiff, aif, au, snd, paf, iff, svx, sf, voc, w64, pvf, xi, htk, caf, sd2, wav, au, aiff, aif, dff, dsf, aac, mpc, wv, mptm, mod, s3m, xm, it, 669, amf, ams, c67, dbm, digi, dmf, dsm, dtm, far, imf, ice, j2b, m15, mdl, med, mms, mt2, mtm, nst, okt, plm, psm, pt36, ptm, sfx, sfx2, st26, stk, stm, stp, ult, wow, gdm, mo3, oxm, umx, xpk, ppm, mmcmp, 669, amf, ams, dbm, dfm, dsm, far, it, med, mdl, mod, mtm, mt2, okt, s3m, stm, ult, umx, xm, amf, dsm, far, gdm, imf, it, med, mod, mtm, s3m, stm, stx, ult, uni, xm, sid, mus, str, prg, P00, mid, mid, 16sv, 3g2, 3gp, 4xm, 8svx, aa3, aac, ac3, adx, afc, aif, aifc, aiff, al, alaw, amr, anim, apc, ape, asf, atrac, au, aud, avi, avm2, avs, bap, bfi, c93, cak, cin, cmv, cpk, daud, dct, divx, dts, dv, dvd, dxa, eac3, film, flac, flc, fli, fll, flx, flv, g726, gsm, gxf, iss, m1v, m2v, m2t, m2ts, m4a, m4b, m4v, mad, mj2, mjpeg, mjpg, mka, mkv, mlp, mm, mmf, mov, mp+, mp1, mp2, mp3, mp4, mpc, mpeg, mpg, mpga, mpp, mpu, mve, mvi, mxf, nc, nsv, nut, nuv, oga, ogm, ogv, ogx, oma, ogg, omg, opus, psp, pva, qcp, qt, r3d, ra, ram, rl2, rm, rmvb, roq, rpl, rvc, shn, smk, snd, sol, son, spx, str, swf, tak, tgi, tgq, tgv, thp, ts, tsp, tta, xa, xvid, uv, uv2, vb, vid, vob, voc, vp6, vmd, wav, webm, wma, wmv, wsaud, wsvga, wv, wve, ay, gbs, gym, hes, kss, nsf, nsfe, rsn, sap, spc, vgm, vgz
  * **mime_type** audio/mpeg, application/ogg, application/x-ogg, audio/ogg, audio/vorbis, audio/vorbis+ogg, audio/x-ogg, audio/x-vorbis, audio/x-vorbis+ogg, application/ogg, application/x-ogg, audio/ogg, audio/x-flac+ogg, audio/x-ogg, application/flac, application/x-flac, audio/flac, audio/x-flac, audio/ogg, application/ogg, audio/opus, audio/wav, audio/aiff, audio/x-wav, audio/x-aiff, audio/wav, audio/aiff, audio/x-wav, audio/x-aiff, application/x-dff, audio/x-dff, audio/x-dsd, application/x-dsf, audio/x-dsf, audio/x-dsd, audio/aac, audio/aacp, audio/x-wavpack, application/flv, application/m4a, application/mp4, application/octet-stream, application/ogg, application/x-ms-wmz, application/x-ms-wmd, application/x-ogg, application/x-shockwave-flash, application/x-shorten, audio/8svx, audio/16sv, audio/aac, audio/aacp, audio/ac3, audio/aiff, audio/amr, audio/basic, audio/flac, audio/m4a, audio/mp4, audio/mpeg, audio/musepack, audio/ogg, audio/opus, audio/qcelp, audio/vorbis, audio/vorbis+ogg, audio/wav, audio/x-8svx, audio/x-16sv, audio/x-aac, audio/x-ac3, audio/x-adx, audio/x-aiff, audio/x-alaw, audio/x-au, audio/x-dca, audio/x-eac3, audio/x-flac, audio/x-gsm, audio/x-mace, audio/x-matroska, audio/x-monkeys-audio, audio/x-mpeg, audio/x-ms-wma, audio/x-ms-wax, audio/x-musepack, audio/x-ogg, audio/x-vorbis, audio/x-vorbis+ogg, audio/x-pn-realaudio, audio/x-pn-multirate-realaudio, audio/x-speex, audio/x-tta, audio/x-voc, audio/x-wav, audio/x-wma, audio/x-wv, video/anim, video/quicktime, video/msvideo, video/ogg, video/theora, video/webm, video/x-dv, video/x-flv, video/x-matroska, video/x-mjpeg, video/x-mpeg, video/x-ms-asf, video/x-msvideo, video/x-ms-wmv, video/x-ms-wvx, video/x-ms-wm, video/x-ms-wmx, video/x-nut, video/x-pva, video/x-theora, video/x-vid, video/x-wmv, video/x-xvid, audio/x-mpd-ffmpeg, audio/L16, audio/L24, audio/x-mpd-float, audio/x-mpd-cdda-pcm, audio/x-mpd-cdda-pcm-reverse, audio/x-mpd-alsa-pcm
* **CONFIG**
  * **rgbs** r=1, g=1, b=1, a=0.75
  * **skin** desktop
  * **music_directory_albumart** /home/media/music
  * **sortbycomposer** true
  * **composergenre** true
  * **composergenrename** 0=Classical
  * **preferlocalfiles** true
  * **mopidy_collection_folders** 0=Local media, 1=SoundCloud/Liked
  * **lastfm_country_code** DE
  * **country_userset** true
  * **debug_enabled** 0
  * **custom_logfile** 
  * **cleanalbumimages** true
  * **do_not_show_prefs** false
  * **load_plugins_at_loadtime** false
  * **beets_server_location** mediacenter.home:8337
  * **dev_mode** false
  * **live_mode** false
  * **use_mopidy_scan** false
  * **collection_load_timeout** 3600000
  * **smartradio_chunksize** 5
  * **linkchecker_nextrun** 1642101958985
  * **linkchecker_isrunning** false
  * **linkchecker_frequency** 604800000
  * **linkchecker_polltime** 5000
  * **audiobook_directory** audiobooks
  * **collection_player** mopidy
  * **snapcast_server** mediacenter.home
  * **snapcast_port** 1705
  * **snapcast_http** 1780
  * **http_port_for_mopidy** 
  * **multihosts** Irene=[host=mediacenter.home, port=6610, password=lollypop, socket=, mopidy_remote=false, radioparams=[radiomode=, radioparam=null, radiomaster=, radioconsume=0]], Daniel=[host=mediacenter.home, port=6600, password=eeree, socket=, mopidy_remote=false, radioparams=[radiomode=, radioparam=null, radiomaster=, radioconsume=0]]
  * **old_style_sql** false
  * **auto_audiobook** 
  * **displaycomposer** true
  * **artistsatstart** 0=Various Artists, 1=Soundtracks
  * **nosortprefixes** 0=The
  * **sortcollectionby** artist
  * **sortresultsby** sameas
  * **actuallysortresultsby** artist
  * **sync_lastfm_playcounts** false
  * **sync_lastfm_at_start** true
  * **next_lastfm_synctime** 0
  * **lastfm_sync_frequency** 86400000
  * **lfm_importer_start_offset** 0
  * **lfm_importer_last_import** 0
  * **hide_master_volume** true
  * **sortbydate** false
  * **notvabydate** false
  * **currenthost** Daniel
  * **player_backend** mpd
  * **collectionrange** 0
  * **lastfm_user** [Redacted]
  * **autotagname** 
  * **lastfm_logged_in** true
  * **tradsearch** false
  * **lastfm_scrobbling** false
  * **lastfm_autocorrect** true
  * **sourceshidden** false
  * **playlisthidden** false
  * **infosource** lastfm
  * **sourceswidthpercent** 25
  * **playlistwidthpercent** 25
  * **downloadart** true
  * **clickmode** double
  * **chooser** albumlist
  * **hide_albumlist** false
  * **hide_filelist** false
  * **hide_radiolist** false
  * **hide_podcastslist** false
  * **hide_playlistslist** false
  * **hide_audiobooklist** false
  * **hide_searcher** false
  * **hidebrowser** false
  * **shownupdatewindow** 1.60
  * **scrolltocurrent** true
  * **alarm_ramptime** 30
  * **alarm_snoozetime** 8
  * **lastfmlang** interface
  * **synctags** true
  * **synclove** true
  * **synclovevalue** 5
  * **theme** Numismatist.css
  * **icontheme** Bobalophagus-Dark
  * **coversize** 40-Large.css
  * **fontsize** 05-Huge.css
  * **fontfamily** Nunito.css
  * **displayresultsas** collection
  * **crossfade_duration** 5
  * **newradiocountry** countries/GB
  * **search_limit_limitsearch** false
  * **scrobblepercent** 50
  * **updateeverytime** false
  * **fullbiobydefault** true
  * **mopidy_search_domains** 0=local, 1=spotify
  * **mopidy_radio_domains** 0=local, 1=spotify
  * **outputsvisible** false
  * **wheelscrollspeed** 150
  * **searchcollectiononly** false
  * **displayremainingtime** true
  * **cdplayermode** false
  * **auto_discovembobulate** false
  * **sleeptime** 30
  * **sleepon** false
  * **sortwishlistby** artist
  * **player_in_titlebar** true
  * **communityradioorderby** name
  * **browser_id** 1640776287760
  * **playlistswipe** true
  * **default_podcast_display_mode** 0
  * **default_podcast_refresh_mode** 4
  * **default_podcast_sort_mode** 0
  * **podcast_mark_new_as_unlistened** false
  * **use_albumart_in_playlist** true
  * **podcast_sort_levels** 4
  * **podcast_sort_0** Title
  * **podcast_sort_1** Artist
  * **podcast_sort_2** Category
  * **podcast_sort_3** new
  * **alarms** 
  * **lastversionchecked** 1.00
  * **lastversionchecktime** 1641498203042
  * **playlistbuttons_isopen** false
  * **collectionbuttons_isopen** false
  * **advsearchoptions_isopen** false
  * **podcastbuttons_isopen** false
  * **last_cache_clean** 1641628717346
  * **next_podcast_refresh** 1641687637364
  * **use_original_releasedate** false
  * **bgimgparms** dummy=baby, Numismatist.css=[timeout=60000, lastchange=1641497157064, random=false, position=top left]
  * **chartoption** 0
  * **consume_workaround** false
  * **we_do_consume** false
  * **last_lastfm_synctime** 1631619549000
  * **mpd_host** mediacenter.home
  * **mpd_port** 6600
  * **mpd_password** 
  * **unix_socket** 
  * **collection_type** sqlite
  * **mopidy_http_port** false
  * **collectioncontrolsvisible** true
  * **hide_pluginplaylistslist** true
  * **maxrgbs** r=150, g=150, b=150, a=0.75
  * **bgdrgbs** r=1, g=1, b=1, a=0
  * **radiomode** 
  * **radioparam** null
  * **radiomaster** 
  * **radioconsume** 0
```

**Comment by fatg3erman** [comment]
Thanks for the detailed logs. I did make a small change in that area for 1.60 and yes, I broke it :)

**Comment by fatg3erman** [comment]
Just released 1.60.1 which fixes this problem. Thanks for reporting.
